IDR/notebook/2.collect_statistics.ipynb

```diff
@@ -1,127 +1,127 @@
-    "stat_results_df.to_parquet(output_file, compression=\"gzip\")"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 3,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Collect databank stats\n",
-    "databank_stats = collect_databank_stats(metadata_dir=studies_metadata_dir, na_cols=[\"pixel_size_x\", \"pizel_size_y\"])\n",
-    "\n",
-    "# Save databank stats as parquet file\n",
-    "output_file = pathlib.Path(stats_dir, f\"databank_diversity.parquet.gzip\")\n",
-    "databank_stats.to_parquet(output_file, compression=\"gzip\")"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 4,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "                              Study_Name       Attribute     S         H  NME  \\\n",
-      "0  idr0080-way-perturbation_screenA_2701         well_id  6912  8.841014  1.0   \n",
-      "1  idr0080-way-perturbation_screenA_2701  imaging_method     1  0.000000  NaN   \n",
-      "2  idr0080-way-perturbation_screenA_2701        organism     1  0.000000  NaN   \n",
-      "3  idr0080-way-perturbation_screenA_2701   organism_part     1  0.000000  NaN   \n",
-      "4  idr0080-way-perturbation_screenA_2701       cell_line     3  1.098612  1.0   \n",
-      "\n",
-      "     J    E   GC  \n",
-      "0  1.0  1.0  0.0  \n",
-      "1  NaN  1.0  0.0  \n",
-      "2  NaN  1.0  0.0  \n",
-      "3  NaN  1.0  0.0  \n",
-      "4  1.0  1.0  0.0  \n",
-      "    Attribute      S          H       NME         J         E        GC\n",
-      "0   screen_id      3   0.860492  0.812077  0.783253  0.674471  0.363842\n",
-      "1  study_name      3   0.860492  0.812077  0.783253  0.674471  0.363842\n",
-      "2    plate_id    334   5.494304  0.330586  0.945478  0.534196  0.371400\n",
-      "3  plate_name    334   5.494304  0.330586  0.945478  0.534196  0.371400\n",
-      "4     well_id  28320  10.251324  1.000000  1.000000  1.000000  0.000000\n"
-     ]
-    }
-   ],
-   "source": [
-    "study_stats = pd.read_parquet(f\"{stats_dir}/individual_studies_diversity.parquet.gzip\")\n",
-    "print(study_stats.head())\n",
-    "\n",
-    "databank_stats = pd.read_parquet(f\"{stats_dir}/databank_diversity.parquet.gzip\")\n",
-    "print(databank_stats.head())"
-   ]
-  }
- ],
- "metadata": {
-  "kernelspec": {
-   "display_name": "Python 3.10.7 64-bit",
-   "language": "python",
-   "name": "python3"
-  },
-  "language_info": {
-   "codemirror_mode": {
-    "name": "ipython",
-    "version": 3
-   },
-   "file_extension": ".py",
-   "mimetype": "text/x-python",
-   "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.10.7"
-  },
-  "orig_nbformat": 4,
-  "vscode": {
-   "interpreter": {
-    "hash": "aee8b7b246df8f9039afb4144a1f6fd8d2ca17a180786b69acc140d282b71a49"
+    "cells": [
```

Comment:
This is an extremely clean notebook, very easy for see what is going on. Well done!

One thing that you might consider adding (which would make this notebook complete) is a brief markdown cell in the beginning of the notebook describing what the notebook does. The documentation can be very brief. For example...

# Collect statistics

Acquire diversity-based statistics for each study and for the databank (IDR) as a whole.
See the repository [README.md](https://github.com/WayScience/publicly-available-microscopy-data#statistics) for full statistics details.

Reply:
Added in fbe29b3f4e1eaa1f29f613da715b428e4c69e99f.